src/open_inwoner/openzaak/models.py

```diff
@@ -46,45 +47,160 @@ def generate_default_file_extensions():
     )
 
 
-class OpenZaakConfig(SingletonModel):
-    """
-    Global configuration and defaults for zaken and catalogi services.
-    """
+class ZGWApiGroupConfig(models.Model):
+    """A set of of ZGW service configurations."""
+
+    open_zaak_config = models.ForeignKey(
+        "openzaak.OpenZaakConfig", on_delete=models.PROTECT, related_name="api_groups"
+    )
```

Comment:
`OpenZaakConfig` is a `SingletonModel` and there will only be one instance of it, so I don't think we need a `ForeignKey` to it on this model and can just use `.get_solo` if we need it?

Reply:
Hmm yeah that's true, I don't feel too strongly about either way, so it's fine